python/tvm/autotvm/tuner/tuner.py

```diff
@@ -174,6 +176,13 @@ def tune(self, n_trial, measure_option, early_stopping=None, callbacks=(), si_pr
             else:
                 logger.setLevel(old_level)
 
+        if error_ct == i:
+            logging.warning(
+                "Could not find a single schedule for task %s. A fallback config will be used. "
+                "The following errors occured:\n  %s",
+                self.task,
+                "\n  ".join(errors),
```

Comment:
This could be super long if you set trial to a large number (e.g., 1,500) and it could easily wipe out the most important message (i.e., no valid schedule found). How about dumping them to a log file and only show the log file name here?

Reply:
Good point. I've switched it to output to a temporary file.